### ISSUE TITLE
Logging in Via Microsoft on Mac

### ISSUE THREAD

**Issue by MadamMars** [killpowa/Twickt-Launcher#1263]
I've been using GDL for a long time but when I downloaded the newest update to support Mojangs migration to Microsoft, it doesn't allow me to log in. 

I know next to nothing about coding and whatnot so here is what I have:
![Screen Shot 2022-01-23 at 2 38 00 PM](https://user-images.githubusercontent.com/98283143/150695243-27cc2fa8-7499-439c-9efa-c9c07a8dc77a.png)

If you need more info, I'll be happy to provide! Thanks!

**Comment by FieryMewtwo** [comment]
Are you a minor? If so, you'll have to change your Xbox DOB. GDL doesn't support age 17- accounts yet.

**Comment by MadamMars** [comment]
> Are you a minor? If so, you'll have to change your Xbox DOB. GDL doesn't support age 17- accounts yet.

No, I’m not a minor! I play on a Mac and it asked something about needing a password to a keychain?? But like I said, this is a little out of my league lol

**Comment by FieryMewtwo** [comment]
Hmmmm... That I'm not sure about.

**Comment by ColdIce1605** [comment]
Ok, glad to see its also affecting someone else, I'm on Linux(EndeavourOS) using the gdlauncher-bin package and I can confirm not a minor.

**Comment by Eskaan** [comment]
Coluld you send your console too? Maybe the services hat an outage (->Network error).

**Comment by ColdIce1605** [comment]
![Screenshot_20220210_074215~2.png](https://user-images.githubusercontent.com/71466513/153420847-335f323f-ed9b-4139-a596-3bc27b0c14a8.png)

Don't think it would be a outage or I'll hear about it

**Comment by Eskaan** [comment]
You already restarted your Launcher, right?

**Comment by MagicaFreak** [comment]
Same issue on Ubuntu 22.04
It works after login in in the 20. try (so sometimes it works but not consistently)
But after restarting the PC i was logged out again.
Full Error Log:
![Bildschirmfoto von 2022-02-10 15-47-00](https://user-images.githubusercontent.com/49316821/153433811-4c0769e6-93e5-4619-bd4f-d85e9e10d0c2.png)

**Comment by ColdIce1605** [comment]
> You already restarted your Launcher, right?

Yes

**Comment by MagicaFreak** [comment]
@ColdIce1605 Did you reinstall?
After reinstalling, everything works again.

**Comment by huantianad** [comment]
Weird, maybe you can try the non bin package

**Comment by Ackmard** [comment]
I was having the same issue a couple minutes ago on windows-portable. I was logged in fine yesterday but when I opened the app today it just brings me to the login screen and I doesn't work. But I just followed this :
>In %appdata%\gdlauncher_next folder replace rChannel file with this one.
https://cdn.discordapp.com/attachments/422172396502515715/795785466933936208/rChannel

It works. Not sure if this can also fix it in Mac/Linux

**Comment by Truhard** [comment]
![image](https://user-images.githubusercontent.com/72954250/153449020-7da18b15-705b-4ff3-979a-0897b7164a55.png)

Getting this error too.

**Comment by ColdIce1605** [comment]
> @ColdIce1605 Did you reinstall?
> After reinstalling, everything works again.

Yes and no,
I installed the gdlauncher package then tried the other package that's precompiled gdlauncher-bin

**Comment by Truhard** [comment]
Downgraded to 1.1.0-beta.2, and I was able to log in. Upgraded back to 1.1.21, and it broke again. This is just a random version I clicked, and it may work on a newer version; I didn't test it.

**Comment by commandhat** [comment]
Successfully logged in on 1.1.21 - took multiple attempts, however.

**Comment by Truhard** [comment]
My log-in worked first try on [1.1.14](https://github.com/gorilla-devs/GDLauncher/releases/tag/v1.1.14), and I can see all my instances too (couldn't on 1.1.0-beta.2). 1.1.14 was the most recent version I could log in first try on.
